src/content.js

```diff
@@ -53,8 +53,8 @@ function groupByRepo(events) {
 		return repos;
 	}, new Map());
 }
-function apply(options, insertionPoint) {
-	const holder = domifyEscape`<div class="ghcf-holder"><i>`;
+function apply(options) {
+	const holder = document.createDocumentFragment();
```

Comment:
Yeah but it's just adding [extra code](https://github.com/gloooom/github-clean-feed/blob/0390edb8631ae078164cdc2eb3eb53ed30ee8056/src/content.js#L169-L180) to do the same thing, please restore